api/src/middleware/publications.js

```diff
@@ -4,7 +4,7 @@
 
 const { body, validationResult } = require('express-validator');
 const axios = require('axios');
-const categoryTypeEnum = require('../config/puppeteer');
+const {categoryTypeEnum} = require('../config/puppeteer');
```

Comment:
This was me refactoring because I noticed the validation wasn't happening correctly